jobs/ci-kubernetes-kubemark-100-gce.env

```diff
@@ -0,0 +1,18 @@
+### job-env
+ENABLE_GARBAGE_COLLECTOR=true
+E2E_NAME=kubemark-100
```

Comment:
TODO!!! - move this to config.json and set in --cluster flag. I definitely wrote this flag and let it slipped :-| (too sleepy to update the PR now, so go dream..